constellations/orion/src/domains/account.go

```diff
@@ -18,6 +18,16 @@ type Account struct {
 	Password     string       `json:"password" db:"password"`
 }
 
+type AccountSum struct {
+	Id           uint         `json:"id"`
+	CreatedAt    time.Time    `json:"-" db:"created_at"`
+	UpdatedAt    time.Time    `json:"-" db:"updated_at"`
+	DeletedAt    sql.NullTime `json:"-" db:"deleted_at"`
+	PrimaryEmail string       `json:"primaryEmail" db:"primary_email"`
+	Password     string       `json:"password" db:"password"`
+	Sum          int          `json:"sum" db:"sum"`
```

Comment:
So we don't need the `db:"sum"` here because this sum never gets written into a database column. It's simply a field just for JSON responses, so we only need the json part.